packages/server/test/e2e/2_cookies_spec.coffee

```diff
@@ -181,45 +212,34 @@ describe "e2e cookies", ->
       httpUrl = "http://#{baseDomain}#{if haveRoot then '' else ":#{httpPort}"}"
       httpsUrl = "https://#{baseDomain}#{if haveRoot then '' else ":#{httpsPort}"}"
 
-      ## once browsers are shipping with the options in FORCED_SAMESITE_ENV as default,
-      ## we can remove this extra test condition
       [
-        [FORCED_SAMESITE_ENV, 'with forced SameSite']
-        [{}, 'without forced SameSite']
+        [httpUrl, false],
+        [httpsUrl, true]
       ].forEach ([
-        processEnv,
-        samesite
+        baseUrl,
+        https
       ]) ->
-        context samesite, ->
-          [
-            [httpUrl, false],
-            [httpsUrl, true]
-          ].forEach ([
-            baseUrl,
-            https
-          ]) ->
-            e2e.it "passes with baseurl: #{baseUrl}", {
-              config: {
-                experimentalGetCookiesSameSite: true
-                baseUrl
-                env: {
-                  baseUrl
-                  expectedDomain: baseDomain
-                  https
-                  httpUrl
-                  httpsUrl
-                  otherUrl
-                  otherHttpsUrl
-                }
-              }
-              processEnv: FORCED_SAMESITE_ENV
```

Comment:
as merged, the PR wasn't even testing both conditions, since this was hard-coded to FORCED_SAMESITE_ENV :facepalm: